wp-theme-2018/shortcodes/epfl_memento/view.php

```diff
@@ -34,22 +34,38 @@
       if (is_null($event->start_date) and is_null($event->end_date)) {
         continue;
       }
-
+      
       $is_first_event = ($count==1);
       $is_just_finished = is_just_finished($event->end_date, $event->end_time);
       $is_inscription_required = is_inscription_required($event->invitation);
+
+      $visual_url = get_visual_url($event, $memento_name);
 ?>
 
 <?php if ($is_first_event and $display_first_event): ?>
 
 <div class="card-slider-cell card-slider-cell-lg">
   <a href="<?php echo esc_url($event->event_url) ?>" class="card card-gray link-trapeze-horizontal">
     <div class="card-body">
-      <?php get_template_part('shortcodes/epfl_memento/templates/card-img-top');  ?>
+      <picture class="card-img-top">
+        <?php if ($event->academic_calendar_category != null): ?>
```

Comment:
Pareil qu'avant, avec un empty(), c'est un plus
```suggestion
        <?php if (!empty($event->academic_calendar_category)): ?>
```